dataplane/docs/libsaibuild.md

```diff
@@ -0,0 +1,14 @@
+# Building libsai.so
+
+## Bazel
+
+Bazel is the preferred way to build Lemming. There are two bazel targets: the first statically links the library, the other dynamically links glog, protobuf, gRPC libraries.
+The second is used to distribute a deb package for use in syncd. In order to ensure shared library version compatibility, bazel executes the build in a seperate docker container.
+The image is not released publicly, but can be built using the [Dockerfile](../standalone/Dockerfile).
+
+1. `bazel build //dataplane/standalone:sai-deb --config docker-bullseye`
+2. Built debian package is located at `bazel-bin/dataplane/standalone`
```

Comment:
I don't quite understand how the bullseye config comes into play here, can you explain that here?

Reply:
done